src/map/status.cpp

```diff
@@ -8838,18 +8875,21 @@ t_tick status_get_sc_def(struct block_list *src, struct block_list *bl, enum sc_
 			tick_def2 = (status->vit + status->dex) * 50;
 			break;
 		case SC_WHITEIMPRISON:
-			if( tick == 5000 ) // 100% on caster
+			if( src == bl ) // 100% on caster
 				break;
-			if( bl->type == BL_PC )
-				tick_def2 = status_get_lv(bl)*20 + status->vit*25 + status->agi*10;
-			else
-				tick_def2 = (status->vit + status->luk)*50;
+			sc_def = status->str * 10 + status_get_lv(bl) * 10 + status->luk * 10;
```

Comment:
iRO wiki confirm that STR indeed resists this status change for players, but claims that for monsters it's still Vit and Luk. Should we check that? Could be that iROwiki is wrong, though.